### ISSUE TITLE
[BUG] Qt5 client PetStore example doesn't compile when using openapi 3 specification

### ISSUE THREAD

**Issue by MartinDelille** [OpenAPITools/openapi-generator#3457]
#### Bug Report Checklist

- [x] Have you provided a full/minimal spec to reproduce the issue?
- [x] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] What's the version of OpenAPI Generator used?
- [x] Have you search for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Bounty to sponsor the fix ([example](https://www.bountysource.com/issues/66123212-javascript-client-produces-a-wrong-object-for-a-string-enum-type-that-is-used-with-ref))

##### Description

As explained [here](https://github.com/OpenAPITools/openapi-generator/pull/3415#issuecomment-514574196), the code generate from the openapi 3 spec of the PetStore example doesn't compile.

##### openapi-generator version

4.0.3

##### OpenAPI declaration file content or url

https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/test/resources/3_0/petstore.yaml

##### Command line used for generation

```
openapi-generator generate -i petstore.yaml -g cpp-qt5-client
```

##### Steps to reproduce

1. Modify the api specification example in `bin/cpp-qt5-petstore.sh` to modules/openapi-generator/src/test/resources/3_0/petstore.yaml
1. Try to compile `samples/client/petstore/cpp-qt5/PetStore/PetStore.pro`

##### Related issues/PRs

- Run Qt5 client sample test #3415
- qt5 client: use openapi 3 spec #3400
- Revert "qt5 client: use openapi 3 spec" #3458

##### Suggest a fix

I find it strange that `OAIInline_object*` file are generated since they are not used anywhere...

**Comment by etherealjoy** [comment]
Issue due to file upload not being handled.

**Comment by etherealjoy** [comment]
solved in #3853
